synthtool/sources/templates.py

```diff
@@ -74,3 +79,43 @@ def render(self, **kwargs) -> Path:
             _render_to_path(self.env, template_name, self.dir, kwargs)
 
         return self.dir
+
+
+#
+# Generates a markdown badge for displaying a "Release Quality'.
+#
+# @param {string} release_quality One of: (ga, beta, alpha, eap, deprecated).
+# @returns {string} The markdown badge.
+def release_quality_badge(input):
+    if not input:
+        log.error(f"ensure you pass a string 'quality' to release_quality_badge")
+        return
+
+    release_quality = input.upper()
+    badge = ""
+
+    if release_quality == "GA":
+        badge = "general%20availability%20%28GA%29-brightgreen"
+    elif release_quality == "BETA":
+        badge = "beta-yellow"
+    elif release_quality == "ALPHA":
```

Comment:
@busunkim96 went ahead and ported the logic from JavaScript; still have some work to do to load samples from disk.